tests/test_ai.py

```diff
@@ -22,15 +22,19 @@ def test_ai_strategies(self):
         n_rounds = 3
         for _ in range(n_rounds):
             for strategy in AI_STRATEGIES:
+                if strategy in [AI_HUMAN, AI_PRO]:
+                    continue
```

Comment:
why cant we test?

Reply:
Something was going wrong. From what I can see, it's forcing the test bench to use the default config, which has no human model specified. At present the user has to manually download the human model (at least by pressing the button) and then link to it in settings. If we don't want to bundle Katrain with the human models automatically, then unfortunately we can't automatically test them, either :(

Reply:
we can look into bundling with the human model if it's not too big.
regardless, we could curl it in the gjthub action, lemme see